api/src/main/java/com/csiro/tickets/controllers/AttachmentController.java

```
@@ -62,6 +87,112 @@ public ResponseEntity<ByteArrayResource> getThumbnail(
     }
   }
 
+  @PostMapping(
+      value = "/api/attachments/upload/{ticketId}",
+      produces = MediaType.APPLICATION_JSON_VALUE)
+  @Transactional
+  public ResponseEntity<AttachmentUploadResponse> uploadAttachment(
+      @PathVariable Long ticketId, @RequestParam("file") MultipartFile file) {
+    if (file.isEmpty()) {
+      return ResponseEntity.badRequest()
```

I think for this we should throw an extension of the SnomioProblem, just so all invalid reqs are in the same format? With that being said I'm not 100% on what format this response is, but it would be good if it was the same.